buildchain/buildchain/packaging.py

```diff
@@ -352,14 +352,11 @@ def _deb_package(name: str, sources: Path) -> targets.DEBPackage:
             'Missing version for package "{}"'.format(name)
         )
 
-    # In case the `release` is of form "{build_id}.{os}", which is standard
-    build_id_str, _, _ = pkg_info.release.partition('.')
-
     return targets.DEBPackage(
         basename='_build_deb_packages',
         name=name,
         version=pkg_info.version,
-        build_id=int(build_id_str),
+        build_id=int(pkg_info.release),
```

Comment:
commit message:
- in the title, s/nuber/number
- in the body, s/gave/given/

Also it's not "to avoid parsing" but "because the release number IS the build id for DEB package that we build ourselve".

Reply:
It's changed